### DISCUSSION TITLE
I have no idea why, but this video reminds me of working on this project

### DISCUSSION THREAD

**Discussion by oleg-derevenetz** [ihhub/fheroes2#5217] [Show and tell]
https://www.youtube.com/watch?v=JmSXo0XdWoA

**Comment by ihhub** [comment]
We're on the same wave😅

**Comment by LeHerosInconnu** [comment]
What a great magician, he knows his magic. :)

Also, you will notice that he does the tests himself. :rofl:

**Comment by Branikolog** [comment]
I'm deeply convinced, that you all, guys, working on our project show even more patience and professional skills, comparing to that nice craftsman from the video. ;)
